huaweicloud/services/gaussdb/resource_huaweicloud_gaussdb_redis_instance.go

```diff
@@ -164,6 +164,14 @@ func ResourceGaussRedisInstanceV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"lb_ip_address": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"lb_port": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
need to set these attributes in the resourceGaussRedisInstanceV3Read func

Reply:
done